hcsctl/Makefile

```diff
@@ -6,7 +6,9 @@ check:
 build:
 	pkger -include /sample-inventory
 	go build -o build/hcsctl
-	cd e2e && ginkgo build && mv e2e.test ../build/hcsctl.test && cd ..
+	cd e2e/cdi && ginkgo build && mv cdi.test ../../build/cdi.test && cd ..
+	cd e2e/ceph && ginkgo build && mv ceph.test ../../build/ceph.test && cd ..
```

Comment:
Since we are using `rook` and `cdi` as sample-inventory folder names, and using `[TEST][Rook]` for each e2e test cases, I think having folder name `rook` under `e2e` directory and `rook.test` as a test binary name is more coherent for users.